jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/internals/lifecycle/MVELLifeCycleManagerTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.jbpm.services.task.internals.lifecycle;
+
+import java.io.StringReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jbpm.services.task.exception.PermissionDeniedException;
+import org.jbpm.services.task.impl.factories.TaskFactory;
+import org.jbpm.services.task.impl.model.UserImpl;
+import org.junit.Test;
+import org.kie.api.task.model.PeopleAssignments;
+import org.kie.api.task.model.Status;
+import org.kie.api.task.model.Task;
+import org.kie.api.task.model.User;
+import org.kie.internal.task.api.model.InternalTask;
+import org.kie.internal.task.api.model.Operation;
+
+public class MVELLifeCycleManagerTest {
```

Comment:
testClaimIsAllowedExcludedOwner test already does that (it makes sure PermissionDeiniedException is thrown). Are you looking for us to add more tests like that?